### ISSUE TITLE
Scheduled service problem

### ISSUE THREAD

**Issue by mjasinsk** [jonnybergdahl/HomeAssistant_Growcube_Integration#7]
Hi,

in my case all chanells have to  have the same value. I thought that every channel could be set separately. And is not. Is there any chane to change?

**Comment by jonnybergdahl** [comment]
Activate logging and post your log.

To activate logging, under Integrations, click the arrow to the right of the Growcube logo. In the left menu, click "Enable debug logging"

**Comment by mjasinsk** [comment]
[home-assistant_growcube_2024-08-03T16-02-55.262Z.log](https://github.com/user-attachments/files/16481153/home-assistant_growcube_2024-08-03T16-02-55.262Z.log)

**Comment by mjasinsk** [comment]
i hope that it is about this file :)

**Comment by jonnybergdahl** [comment]
Yes, it looks just fine to me. 

```2024-08-03 18:02:42.634 DEBUG (MainThread) [custom_components.growcube.coordinator] growcube_259f59: Sensor reading, channel Channel.Channel_D, humidity 41, temperature 27, moisture 38
2024-08-03 18:02:43.727 DEBUG (MainThread) [custom_components.growcube.coordinator] growcube_259f59: Water state False
2024-08-03 18:02:43.782 DEBUG (MainThread) [custom_components.growcube.coordinator] growcube_259f59: Sensor reading, channel Channel.Channel_A, humidity 41, temperature 27, moisture 37
2024-08-03 18:02:43.841 DEBUG (MainThread) [custom_components.growcube.coordinator] growcube_259f59: Sensor reading, channel Channel.Channel_B, humidity 41, temperature 27, moisture 21
2024-08-03 18:02:43.902 DEBUG (MainThread) [custom_components.growcube.coordinator] growcube_259f59: Sensor reading, channel Channel.Channel_C, humidity 41, temperature 27, moisture 44```

Moisture A = 37
Moisture B = 21
Moisture C = 44
Moisture D = 38

All different values.

**Comment by jonnybergdahl** [comment]
Not sure what you are asking. You set the channel like this, once for each channel. 
![Skärmavbild 2024-08-04 kl  18 47 49](https://github.com/user-attachments/assets/ad061ed0-da3f-4d06-9ee2-1cde857e1daf)

**Comment by mjasinsk** [comment]
I tried set valus in different way for each and every channel in the way i can do it in grow cube app. Channel A and B moisture 30 -50  and all day watering channel C 20-60 after sunset watering chanel D 40-60  a.s.  watering. It seems that this doesn't work in such a way a s in original appBRMaciekWysłano z telefonu Samsung
-------- Oryginalna wiadomość --------Od: Jonny Bergdahl ***@***.***> Data: 4.08.2024  18:49  (GMT+01:00) Do: jonnybergdahl/HomeAssistant_Growcube_Integration ***@***.***> DW: mjasinsk ***@***.***>, Author ***@***.***> Temat: Re: [jonnybergdahl/HomeAssistant_Growcube_Integration] Scheduled service problem (Issue #7) 
Not sure what you are asking. You set the channel like this, once for each channel.
Skarmavbild.2024-08-04.kl.18.47.49.png (view on web)

**Comment by mjasinsk** [comment]
Hi,

I observed GC this week and you right everything works. I wrongly recognized how Services in HA works and i thought that after setting A on specific value an dthe setting B, when I return to A previously set value will be displayed. Nevertheless it works. I set A for All day watering B for Evening watering and it works. True hell it was to setup GC to work with wireless network 🙂 Thx for help

regards

Maciek

pozdrawiam

Maciek

***@***.***
+48 600 367136
________________________________
Od: Jonny Bergdahl ***@***.***>
Wysłane: niedziela, 4 sierpnia 2024 18:49
Do: jonnybergdahl/HomeAssistant_Growcube_Integration ***@***.***>
DW: mjasinsk ***@***.***>; Author ***@***.***>
Temat: Re: [jonnybergdahl/HomeAssistant_Growcube_Integration] Scheduled service problem (Issue #7)


Not sure what you are asking. You set the channel like this, once for each channel.
Skarmavbild.2024-08-04.kl.18.47.49.png (view on web)<https://github.com/user-attachments/assets/ad061ed0-da3f-4d06-9ee2-1cde857e1daf>

**Comment by jonnybergdahl** [comment]
Oh, yes. Unfortunately there is no way of reading the settings, so if you set it with Home Assistant the phone app has no idea what you set it to.

**Comment by jonnybergdahl** [comment]
Close as it works as intended

**Comment by mjasinsk** [comment]
Yes, once again thank you for helpBest regardsMaciekWysłano z telefonu Samsung
-------- Oryginalna wiadomość --------Od: Jonny Bergdahl ***@***.***> Data: 12.08.2024  13:09  (GMT+01:00) Do: jonnybergdahl/HomeAssistant_Growcube_Integration ***@***.***> DW: mjasinsk ***@***.***>, Author ***@***.***> Temat: Re: [jonnybergdahl/HomeAssistant_Growcube_Integration] Scheduled service problem (Issue #7) 
Closed #7 as completed.
